oryx-app-mllib/src/main/java/com/cloudera/oryx/app/mllib/kmeans/KMeansEvaluation.java

```diff
@@ -97,7 +99,10 @@ private double calcDBIndex(Map<Integer, Tuple2<Double, Long>> clusterSumDistAndC
       totalDBIndex += maxDBIndex;
     }
 
-    return (1.0 / (totalDBIndex / clusters.size()));
+    double daviesBouldinIndex = totalDBIndex / numClusters;
+    log.info("Computed Davies-Bouldin Index for {} clusters: {}", numClusters, daviesBouldinIndex);
+
+    return (1.0 / daviesBouldinIndex);
```

Comment:
I think the inversion can happen in the caller. Right now this method isn't actually returning the DB index as it sort of says it does. I'll make a few changes to make this a little more consistent.